readme.md

```diff
@@ -1,1287 +1,114 @@
-	StripeInvoice response = invoiceService.Pay(*invoiceId*);
-```
-
-### List all invoices
-
-```csharp
-	var invoiceService = new StripeInvoiceService();
-	IEnumerable<StripeInvoice> response = invoiceService.List(); // optional StripeInvoiceListOptions
-```
-
-StripeInvoiceListOptions supports a CustomerId, [StripeListOptions](#stripelistoptions-paging) for paging, and a [StripeDateFilter](#stripedatefilter-date-filtering) for date filtering
-
-Invoice Items
--------------
-
-### Creating an invoice item
-
-Any invoice items you create for a customer will be added to their bill.
-
-```csharp
-	var myItem = new StripeInvoiceItemCreateOptions();
-	myItem.Amount = 1000;
-	myItem.Currency = "usd";            // "usd" only supported right now
-	myItem.CustomerId = *customerId*;
-	myItem.Description = "na";          // not required
-
-	var invoiceItemService = new StripeInvoiceItemService();
-	StripeInvoiceItem response = invoiceItemService.Create(myItem);
-```
-
-### Retrieving an invoice item
-
-```csharp
-	var invoiceItemService = new StripeInvoiceItemService();
-	StripeInvoiceItem response = invoiceItemService.Get(*invoiceItemId*);
-```
-
-### Updating an invoice item
-
-```csharp
-	var myUpdatedItem = new StripeInvoiceItemUpdateOptions();
-	myUpdatedItem.Amount = 1010;
-	myUpdatedItem.Currency = "usd";        // "usd" only supported right now
-	myUpdatedItem.Description = "test";    // not required
-
-	var invoiceItemService = new StripeInvoiceItemService();
-	StripeInvoiceItem response = invoiceItemService.Update(*invoiceItemId*, myUpdatedItem);
-```
-
-### Deleting an invoice item
-
-```csharp
-	var invoiceItemService = new StripeInvoiceItemService();
-	invoiceItemService.Delete(*invoiceItemId*);
 ```
-
-### List all invoice items
-
-```csharp
-	var invoiceItemService = new StripeInvoiceItemService();
-	IEnumerable<StripeInvoiceItem> response = invoiceItemService.List(); // optional StripeInvoiceItemListOptions
+nuget install Stripe.net
 ```
 
-StripeInvoiceItemListOptions supports a CustomerId, [StripeListOptions](#stripelistoptions-paging) for paging, and a [StripeDateFilter](#stripedatefilter-date-filtering) for date filtering
-
-Metadata
---------
-
-Various calls allow attaching metadata to them such as [Creating a Customer](#creating-a-customer) and [Creating a Charge](#creating-a-charge).
-
-To delete metadata you can call the update operation and pass an empty string (`String.Empty`) or `null` for each value you want to delete.
-
-Payouts
--------
-
-### Creating a payout
-
-This will create a payout to the account you setup when activating your Stripe account.
+From within Visual Studio:
 
-```csharp
-	var payout = new StripePayoutService(*apikey*).Create(
-		new StripePayoutCreateOptions
-		{
-			Amount = 1000,
-			Currency = "usd"
-		}
-	);
-```
+1. Open the Solution Explorer.
+2. Right-click on a project within your solution.
+3. Click on "Manage NuGet Packages..."
+4. Click on the "Browse"-tab and search for "Stripe.net".
```

Comment:
I don't think it makes sense that Stripe.net would be italicized and not in quotes.  The browse + manage nuget packages for sure though.